### PR TITLE
goreplay: init at 1.1.0

### PR DESCRIPTION
**Pull request by lovek323**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by davidak** [comment]
Please follow [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md)! commit message, description

**Comment by lovek323** [comment]
> Please follow CONTRIBUTING.md! commit message, description

Thanks, will update.

File: pkgs/tools/networking/goreplay/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildGoPackage, fetchgit, libpcap }:
+
+buildGoPackage rec {
+  pname = "goreplay";
+  version = "1.1.0";
+  rev = "v${version}";
+
+  goPackagePath = "github.com/buger/goreplay";
+
+  src = fetchgit {
```
Comment (zowoq) [inline pkgs/tools/networking/goreplay/default.nix:10]: Please use `FetchFromGithub`.

File: pkgs/tools/networking/goreplay/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    inherit rev;
+    url = "https://github.com/buger/goreplay.git";
+    sha256 = "07nsrx5hwmk6l8bqp48gqk40i9bxf0g4fbmpqbngx6j5f7lpbk2n";
+  };
+
+  goDeps = ./deps.nix;
```
Comment (zowoq) [inline pkgs/tools/networking/goreplay/default.nix:16]: This isn't needed and the deps.nix file can be removed.

File: pkgs/tools/networking/goreplay/default.nix (hunk 1 above)
Comment (lovek323) [reply pkgs/tools/networking/goreplay/default.nix]: Updated. Thanks for reviewing.

**Pull request merged**